actions/class.PropertiesAuthoring.php

```diff
@@ -304,6 +304,29 @@ public function getClassForm(core_kernel_classes_Class $class): tao_helpers_form
                 //save all properties values
                 if (isset($data['properties'])) {
                     $this->saveProperties($data);
+                    
+                    $elementRangeArray = [];
+                    $groups = $myForm->getGroups();
+                    if (isset($data['properties'])) {
+                        foreach ($data['properties'] as $prop) {
+                            if ($prop['range'] != "") {
+                                $index = strstr($groups['property_' . $prop["uri"]]['elements'][0], '_', true);
+                                $elementRangeArray[$index . "_range_list"] = $prop["range"];
+                            }
+                        }
+                    }
+                    
+                    foreach ($myForm->getElements() as $element) {
+                        if (
+                            $element instanceof tao_helpers_form_elements_xhtml_Combobox &&
+                            array_key_exists($element->getName(), $elementRangeArray)
```

Comment:
Just a suggestion. Not necessary to fix it 🙂 
```suggestion
                            $element instanceof tao_helpers_form_elements_xhtml_Combobox
                            && array_key_exists($element->getName(), $elementRangeArray)
```